### ISSUE TITLE
Accessibility - Changes in Headers

### ISSUE THREAD

**Issue by IAmTamal** [IAmTamal/Milan#297]
### Description

 - Take a look at the web app, you'll find that we are using **alot of h1, h2 and so on**
 - Make sure we use it in a proper manner. 
 - Like in a div, if there are 2 headers we should have **H1 first, and then H2** , not **H3, then H1 and so on**
 - Make sure the font sizes remains the same, you just need to make changes to the tag ! 

## Here's what i am talking about  
![image](https://user-images.githubusercontent.com/72851613/190562556-e79f9565-43b5-43f3-a47a-6f9ef48c9b81.png)
![image](https://user-images.githubusercontent.com/72851613/190562584-f864d3da-f3d3-45ba-ae71-08a5adb69230.png)

- As you can see we have **H3 first, H1 second** - we should not have these !

**Comment by 4k5h1t** [comment]
Hey, I think I can take care of this issue. Could you assign it to me ?

**Comment by IAmTamal** [comment]
> Hey, I think I can take care of this issue. Could you assign it to me ? Or if this is in prep for Hacktoberfest, ill take it then :D

Yes this is infact a prep for Hf, i want the web-site to be all good before actually we add alot of features during the event !

**Comment by 4k5h1t** [comment]
> > Hey, I think I can take care of this issue. Could you assign it to me ? Or if this is in prep for Hacktoberfest, ill take it then :D
> 
> Yes this is infact a prep for Hf, i want the web-site to be all good before actually we add alot of features during the event !

Awesome, ill get it done then

**Comment by 4k5h1t** [comment]
Btw, just checking a few things, you have just one h2 tag in a bunch of files, do they need to be h1 or was that a design choice ?
Also, in places with h3 then h6, dyou want it to go to h1 and h2, h1 and h3, h3 and h4, or stay the same ?

**Comment by IAmTamal** [comment]
> 

I want everything to be in an ascending order. 
Like if a file is full of h5, make it all h1. 

IF a div has h2 h6  make it h1 h2 and so on.

**Comment by 4k5h1t** [comment]
Ook got it, few remaining then

**Comment by minhaj-313** [comment]
Hi, I want to contribute to your projects, Can you accept my PR, I want to start my journey in Hacktoberfest, you can see my portfolio website as well :[https://minhaj-313.github.io/My-Portfolio-Website/] My Portfolio Website

**Comment by IAmTamal** [comment]
> Hi, I want to contribute to your projects, Can you accept my PR, I want to start my journey in Hacktoberfest, you can see my portfolio website as well :[https://minhaj-313.github.io/My-Portfolio-Website/] My Portfolio Website

 - You have not created a PR yet
 - I donot need your Portfolio for Hacktoberfest, it's for everyone
 - You can make new issues, if you want to.
